src/main/java/com/mesabrook/ib/blocks/gui/telecom/GuiPhoneCrashed.java

```diff
@@ -111,22 +116,26 @@ protected void actionPerformed(GuiButton button) throws IOException
         if(button == resetButton)
         {
             isPhoneUnlocked = false;
-            ClientSoundPacket soundPacket = new ClientSoundPacket();
-            soundPacket.pos = Minecraft.getMinecraft().player.getPosition();
-            soundPacket.soundName = "minedroid_startup";
-            PacketHandler.INSTANCE.sendToServer(soundPacket);
-            Minecraft.getMinecraft().displayGuiScreen(new GuiLockScreen(phoneStack, hand));
+            Minecraft.getMinecraft().displayGuiScreen(new GuiMSACBootScreen(phoneStack, hand));
         }
 
-        if(button == copyStackToClipboardButton)
+        if(button == uploadToPastebinButton)
         {
-            StringSelection stringSelection = new StringSelection(getErrorStackTrace());
-            Clipboard clipboard = Toolkit.getDefaultToolkit().getSystemClipboard();
-
-            clipboard.setContents(stringSelection, null);
-            copyStackToClipboardButton.enabled = false;
-
-            Toaster.forPhoneNumber(phoneStackData.getPhoneNumberString()).queueToast(new Toast("Error Copied"));
+            try
+            {
+                String stackToUpload = errorStackTrace;
+                String PastebinURL = MinedroidCrashLogUploader.uploadText(stackToUpload);
```

Comment:
Let's try to keep the casing of variable names the same...in this case, `pastebinURL`

Reply:
Fixed https://github.com/Mesabrook-Development-Team/Immersibrook/commit/f650b06e6fbb624e078472b42ea47c9ee9e17e9c